package/CHANGELOG

```diff
@@ -98,6 +98,7 @@ Fixes
   * Contact Analysis class respects PBC (Issue #2368)
 
 Enhancements
+  * Added computation of Mean Squared Displacements (#2438, PR #2619)
```

Comment:
I am not sure why this still diffs against the old CHANGELOG. This should now show up under 2.0.0 where you have to add your name to the 2.0.0 author list.